scaffolds/streamlit_on_cloudrun/image_classification/train_and_deploy.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>Add a line in markdown to optionally attach an accelerator for training. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Make sense. Added! Thanks!</p>